src/main/java/com/intuit/graphql/filter/visitors/MongoCriteriaExpressionVisitor.java

```diff
@@ -0,0 +1,221 @@
+/*
+  Copyright 2020 Intuit Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.intuit.graphql.filter.visitors;
+
+import com.intuit.graphql.filter.ast.*;
```

Comment:
Lets replace the * import with specific import statement.

Reply:
Its done.